### ISSUE TITLE
What is the correct way of reseting a timer?

### ISSUE THREAD

**Issue by paulkass** [MickCrozier/timer-stopwatch#6]
Hi,
I have a timer and I would like to reset it in the `done` callback. Adding `timer.reset(10000)` resets the timer, but it does not start it up again.

```
    timer.reset(10000)
    timer.start()
```

 doesn't do the job either. What is the "correct" way of resetting and starting the timer up again?

BTW, starting the timer after resetting it gives this object:

```
{ runTimer: true,
  hasBeenStopped: false,
  stoptime: 0,
  refreshTimer: 
   { _idleTimeout: 1000,
     _idlePrev: 
      { '0': [Function: listOnTimeout],
        _idleNext: [Circular],
        _idlePrev: [Circular],
        msecs: 1000 },
     _idleNext: 
      { '0': [Function: listOnTimeout],
        _idleNext: [Circular],
        _idlePrev: [Circular],
        msecs: 1000 },
     _idleStart: 299712638,
     _onTimeout: [Function],
     _repeat: false },
  done: false,
  almostDoneFired: false,
  doneFired: false,
  countDownMS: 10000,
  ms: 10000,
  refreshRateMS: 1000,
  almostDoneMS: 10000,
  _events: { time: [Function], done: [Function] } }
```

but the `ms` property doesn't change.

**Comment by catalinmiron** [comment]
`timer.stop(); timer.reset(ms); timer.start();` should be fine

**Comment by paulkass** [comment]
Fixed with 9af72498ec963f08d9c3bba1bdd05d27b9b6517c.
